src/cromshell/utilities/cromshellconfig.py

```diff
@@ -141,14 +157,45 @@ def __get_config_dir():
     return config_path
 
 
-def __get_submission_file(config_directory, sub_file_name):
-    """Get File Path To Cromshell Submission File"""
+def get_submission_file_headers(
```

Comment:
@SHuang-Broad , I add this function to return all the headers from the subclasses of WorkflowDatabaseColumns. However, it returns the headers in the order of the subclasses that are written. For example, the Immutable subclass(line75) is before the Mutable subclass(line82) so this function returns the Immutable values before the mutable values. I'm wondering if this makes it vulnerable to breaking.

Another way to get the values is the use the subclasses directly :
```
       mutable_headers = [
            column.value for column in cromshellconfig.MutableSubmissionFileHeader
        ]
        immutable_headers = [
            column.value for column in cromshellconfig.ImmutableSubmissionFileHeader
        ]
        all_headers = immutable_headers + mutable_headers
```
In this scenrio we may not even need WorkflowDatabaseColumns top class

Reply:
Let's talk about this when we meet